rules/integrations/aws/persistence_iam_create_user_via_assumed_role_on_ec2_instance.toml

```
@@ -96,6 +96,19 @@ event.dataset: "aws.cloudtrail"
     and aws.cloudtrail.user_identity.arn: *i-*
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "event.action",
+    "event.outcome",
+    "cloud.region",
+    "related.user",
+    "user_agent.original",
+    "user.name",
+    "aws.cloudtrail.request_parameters",
+    "aws.cloudtrail.response_elements",
+    "aws.cloudtrail.user_identity.arn",
+    "aws.cloudtrail.user_identity.type"
```

```suggestion
    "aws.cloudtrail.user_identity.type",
    "source.address"
```
may be good to pivot in timeline on source.address

---

Added!